src/app/vault/ciphers.component.html

```diff
@@ -88,4 +93,4 @@
                 <i class="fa fa-plus fa-fw"></i>{{'addItem' | i18n}}</button>
         </ng-container>
     </div>
-</ng-container>
+</ng-container>
```

Comment:
Why does this happen? I noticed it on another file too. Removing the empty new line at EOF?